helm/examples/azure/sa/create-sa.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+WORK_DIR=$(dirname $0)
+source ${WORK_DIR}/env-sa
+source ${WORK_DIR}/common-libs
+
+if [[ -z "$SA_RESOURCE_GROUP" ]] || [[ -z "$SA_LOCATION" ]] || [[ -z "$STORAGE_ACCOUNT_NAME" ]]
+then
+   echo "Miss one of SA_RESOURCE_GROUP, SA_LOCATION and STORAGE_ACCOUNT_NAME"
+   exit 1
+fi
+
+[[ -n "$SUBSCRIPTION" ]] && az account set --subscription $SUBSCRIPTION
+
+VALUES_FILE=${WORK_DIR}/../hpcc-azurefile/values.yaml
+SECRET_NAME_INPUT=$(cat $VALUES_FILE | grep "^[[:space:]]*secretName:"|cut -d':' -f2|sed 's/[[:space:]\"]//g')
+SECRET_NAMESPACE_INPUT=$(cat $VALUES_FILE | grep "^[[:space:]]*secretNamespace:"|cut -d':' -f2|sed 's/[[:space:]\"]|\"//g')
```

Comment:
In 's/[[:space:]\\"]|\\"//g' I think the final |\\" is redundant.

Reply:
fixed